experiments/fgo_nn_splines.py

```diff
@@ -227,8 +266,7 @@ def fgo_nn_splines_train_loop(train_dataloader, val_dataloader=None, subseq_len
     chi2_errors = []
     learning_rates = []
     
-    trajectories_to_save = 3
-    results['num_val_traj'] = trajectories_to_save
+
 
     tensorboard_path = os.path.join(output_path, "tensorboard")
```

Comment:
I think we better have all tensorboard logs in one folder for the same group of experiments.

for example, if I run several experiments with splines will it be possible to view all of them and only them on tensorboard?  which command I should run?


maybe out folder should have structure:
./out/splines/tensorboard/experiment_1/events.out....
./out/splines/experiment_1/trajectories ... dataloaders  + checkpoints